tests/test_interpolation_cubic.py

```diff
@@ -0,0 +1,269 @@
+import numpy as np
```

Comment:
Would it be worth having a test to check the convergence of the cubic interpolator? Not sure what convergence would be expected for the sinc one

Reply:
It isn't as I pointed out above

Reply:
Maybe I'm just being stupid and missing it, but where?

Reply:
Sorry completely missed what you were talking about only saw the `numpy`.

Convergence may be tricky, it's usually easier to just do an analytical check where you set the field to polynomial values and check the interpolation is exact.